Cargo.lock

```diff
@@ -165,7 +165,7 @@ dependencies = [
  "bitflags 2.5.0",
  "cexpr",
  "clang-sys",
- "itertools",
+ "itertools 0.12.1",
```

Comment:
This still lists the version that you're trying to avoid. Is it possible that this is unintentional?

Reply:
It's intentional because `itertools` is a transitive dependency in this case (`bindgen` requires an older version). And it's the reason for adding `itertools` to the skip list in `deny.toml`.